nautobot/ui/src/components/apps/__tests__/AppFullWidthComponents.test.js

```diff
@@ -2,7 +2,7 @@ import { render } from "@testing-library/react";
 import AppFullWidthComponentsWithProps from "../AppFullWidthComponents";
 
 // Set NautobotApps to {}; since we are not testing for plugin integration with nautobot
-jest.mock("src/app_imports", () => ({
+jest.mock("../../../app_imports", () => ({
```

Comment:
What is the meaning of replacing `src` with `../../..`? Same comment for similar changes in other files.

Edited: Wait does jest prefer relative path? I saw that in the PR description just now.